docker/scripts/init_bookie.sh

```diff
@@ -19,65 +19,57 @@
 # * See the License for the specific language governing permissions and
 # * limitations under the License.
 # */
-
 source ${SCRIPTS_DIR}/common.sh
 
 function wait_for_zookeeper() {
     echo "wait for zookeeper"
-    until /opt/bookkeeper/bin/bookkeeper org.apache.zookeeper.ZooKeeperMain -server ${BK_zkServers} ls /; do sleep 5; done
+    until zk-shell --run-once "ls /" ${BK_zkServers}; do sleep 5; done
```

Comment:
In order to fix the integration tests I guess we should use the full path of zk-shell.
In fact the previous line used the full path

@dmercuriali can you please help fixing this patch fro @ravisharda  ?